.github/workflows/ci.yml

```diff
@@ -121,9 +132,7 @@ jobs:
       - run: |
           julia --project=docs -e '
             using Pkg
-            Pkg.develop(PackageSpec(path=pwd(), subdir="KomaMRIFiles"))
-            Pkg.develop(PackageSpec(path=pwd(), subdir="KomaMRICore"))
-            Pkg.develop(PackageSpec(path=pwd(), subdir="KomaMRIPlots"))
+            Pkg.develop([PackageSpec(path=pwd(), subdir="."), PackageSpec(path=pwd(), subdir="KomaMRIBase"), PackageSpec(path=pwd(), subdir="KomaMRICore"), PackageSpec(path=pwd(), subdir="KomaMRIFiles"), PackageSpec(path=pwd(), subdir="KomaMRIPlots")])
```

Comment:
```suggestion
            Pkg.develop([
            PackageSpec(path=pwd(), subdir="."), 
            PackageSpec(path=pwd(), subdir="KomaMRIBase"), 
            PackageSpec(path=pwd(), subdir="KomaMRICore"), 
            PackageSpec(path=pwd(), subdir="KomaMRIFiles"), 
            PackageSpec(path=pwd(), subdir="KomaMRIPlots")
            ])
```

Reply:
Is `PackageSpec(path=pwd(), subdir="."), ` necessary?

Reply:
Addessed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/e359294113fe19888ed7824ec9bbd2a337d641a2

Reply:
Is not the same, see indentation and `])` in the end.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/2f6350318cb110e7c694bdd4af3bbc5ac55273de